src/vorta/store/connection.py

```diff
@@ -62,18 +62,18 @@ def init_db(con=None):
     # Delete old log entries after 6 months.
     # The last `create` command of each profile must not be deleted
     # since the scheduler uses it to determine the last backup time.
-    last_backups_per_profile = (
+    last_backups_per_profile = Tuple(
         EventLogModel.select(EventLogModel.profile, fn.MAX(EventLogModel.start_time))
         .where(EventLogModel.subcommand == 'create')
         .group_by(EventLogModel.profile)
     )
-    last_scheduled_backups_per_profile = (
+    last_scheduled_backups_per_profile = Tuple(
         EventLogModel.select(EventLogModel.profile, fn.MAX(EventLogModel.start_time))
         .where(EventLogModel.subcommand == 'create', EventLogModel.category == 'scheduled')
         .group_by(EventLogModel.profile)
     )
 
-    three_months_ago = datetime.now() - timedelta(days=6 * 30)
+    three_months_ago: datetime = datetime.now() - timedelta(days=6 * 30)
```

Comment:
For local variables static type linters are usually able to infer the type themselves.

Reply:
Yup. Those changes might be a bit redundant, but I did those just to improve readability bit.

Reply:
So, should I remove these type annotations or, remove them? I thought that adding them to the whole codebase might make it messy so, I went for only local variables of some methods and functions, and leaving variables that variables that hold some `Field` (as inside modals).